### ISSUE TITLE
Xenial Install

### ISSUE THREAD

**Issue by 1415926535** [bharadwaj-raju/TextSuggest#38]
I'm new to Ubuntu but can run headless LEMP stacks in Azure and have yet to find something I couldn't install. I have all three dependencies on my machine but when I download and install the .deb, I don't know what to do next. I manually created ~/.config/textsuggest/Custom_Words.txt. 

Dumb it down for us idiots please. I used AHK for years in the Winderz world and this is the closest thing I've found that at least does simple text expansion. 

Thanks for your work and time!

**Comment by bharadwaj-raju** [comment]
Simply use your desktop environment's keyboard shortcut utility to attach a shortcut of your choice to the command `textsuggest --auto-selection`. For example, in Unity (Ubuntu's default), you'll find it in System Settings → Keyboard → Custom Shortcuts (or something like that).

**Comment by bharadwaj-raju** [comment]
@1415926535 Is it OK now, or do you want further explanation?

**Comment by 1415926535** [comment]
I'm cool. Thanks. 
Sorry about lag. I forgot about fighting this that day.
